locust_plugins/listeners/cloudwatch.py

```diff
@@ -0,0 +1,172 @@
+import datetime
+import logging
+from gevent import queue
+
+log = logging.getLogger(__name__)
+
+
+class RequestResult:
+    STATUS_SUCCESS = "SUCCESS"
+
+    STATUS_FAILURE = "FAILURE"
+
+    def __init__(self, host, request_type, name, response_time, response_length, exception):
+        self.timestamp = datetime.datetime.utcnow()
```

Comment:
shouldnt we use the start_time from the request event? like in Timescale listener

Reply:
I was working with an earlier version of locust. Will change this.

Reply:
Have changed it